src/admin/class-locations.php

```diff
@@ -377,29 +384,73 @@ function () use ( $post_id ) {
 			}
 		}
 
-		// Make the geometry object.
-		$geometry = [];
-		if ( isset( $_POST['field_latitude'] ) && isset( $_POST['field_longitude'] ) ) {
-			$latitude  = sanitize_text_field( wp_unslash( $_POST['field_latitude'] ) );
-			$longitude = sanitize_text_field( wp_unslash( $_POST['field_longitude'] ) );
-			$geometry  = [
-				'type'        => 'Point',
-				'coordinates' => [ (float) $longitude, (float) $latitude ],
-			];
+		// Execute the save_geometry_object function from the Openkaarten_Base_Functions class.
+		\Openkaarten_Base_Functions\Openkaarten_Base_Functions::save_geometry_object( $post_id, $properties );
+	}
+
+	/**
+	 * Get latitude and longitude from an address with OpenStreetMap.
+	 *
+	 * @param string $address The address.
+	 *
+	 * @return array|null
+	 */
+	public static function convert_address_to_latlong( $address ) {
```

Comment:
Je voegt deze functie in deze PR toe, maar ik zie hem nergens gebruikt worden?

Reply:
Deze functie is uiteindelijk verplaatst naar de Base_Functions plugin en kan hier dus weg inderdaad.